tests/integration/test_db_admin.py

```diff
@@ -153,10 +158,27 @@ async def test_landscape_scalable_bundle_db(ops_test: OpsTest, charm: str) -> No
     except HTTPError as e:
         assert False, f"error when trying to create role on Landscape: {e}"
 
+    database_unit_name = ops_test.model.applications[DATABASE_APP_NAME].units[0].name
+    connection_string = await build_connection_string(
+        ops_test, LANDSCAPE_APP_NAME, RELATION_NAME, remote_unit_name=database_unit_name
+    )
+
     # Remove the applications from the bundle.
     await ops_test.model.remove_application(LANDSCAPE_APP_NAME, block_until_done=True)
     await ops_test.model.remove_application(HAPROXY_APP_NAME, block_until_done=True)
     await ops_test.model.remove_application(RABBITMQ_APP_NAME, block_until_done=True)
 
+    # Remove the relation and test that its user was deleted
+    # (by checking that the connection string doesn't work anymore).
+    async with ops_test.fast_forward():
+        await ops_test.model.applications[DATABASE_APP_NAME].remove_relation(
+            f"{DATABASE_APP_NAME}:{RELATION_NAME}", f"{LANDSCAPE_APP_NAME}:{RELATION_NAME}"
```

Comment:
```suggestion
           f"{DATABASE_APP_NAME}:{RELATION_NAME}", f"{LANDSCAPE_APP_NAME}:db"
```
The interface on Landscape's side is called `db`.

Reply:
Relation name is `db-admin` but the interface on Landscape side is called `db`